src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/FritekstVedtakbegrunnelser.tsx

```diff
@@ -0,0 +1,170 @@
+`;
+
+const StyledFamilieFritekstFelt = styled.div`
+    display: flex;
+    margin-top: 3px;
+    margin-bottom: 3px;
+
+    .textarea__container {
+        width: 100% !important;
+    }
+`;
+
+const StyledElement = styled(Element)`
+    margin-bottom: 13px;
+`;
+
+const UtførKnapp = styled(IkonKnapp)`
+    margin-top: 0px;
+`;
+
+const SletteKnapp = styled(IkonKnapp)`
+    margin-top: 0px;
+    margin-right: 0px;
+`;
+
+const Knapperad = styled.div`
+    display: grid;
+    column-gap: 1rem;
+    grid-template-columns: 100px 100px;
+    margin-top: 24px;
+`;
+
+const FritekstVedtakbegrunnelser: React.FC<IProps> = () => {
+    const { erLesevisning } = useBehandling();
+    const {
+        fritekster,
+        redigerbarefritekster,
+        settRedigerbarefritekster,
+        leggTilRedigerbareFritekst,
+        idPaSistOpprettetFritekst,
+        onSubmit,
+        toggleForm,
+    } = useFritekstVedtakBegrunnelser();
+
+    const harFritekster =
+        Object.keys(redigerbarefritekster).length > 0 || Object.keys(fritekster).length > 0;
+
+    useEffect(() => {
+        const element = document.getElementById(`${idPaSistOpprettetFritekst}`);
+        if (element) {
+            element.focus();
+        }
+    }, [idPaSistOpprettetFritekst]);
+
+    return (
+        <StyledSkjemaGruppe>
+            {harFritekster && (
+                <StyledElement>Fritekst til kulepunkt i brev (valgfri)</StyledElement>
+            )}
+            {Object.keys(redigerbarefritekster).map((fritekstId: string) => {
+                return (
+                    <StyledFamilieFritekstFelt>
+                        <FamilieTextareaBegrunnelseFritekst
+                            tekstLesevisning={''}
+                            erLesevisning={erLesevisning()}
+                            defaultValue={redigerbarefritekster[fritekstId].verdi}
+                            key={`__fritekst-${fritekstId}`}
+                            id={`${fritekstId}`}
+                            textareaClass={'fritekst-textarea'}
+                            value={redigerbarefritekster[fritekstId].verdi}
+                            maxLength={300}
+                            onBlur={(event: React.FocusEvent<HTMLTextAreaElement>) => {
+                                settRedigerbarefritekster({
+                                    ...redigerbarefritekster,
+                                    [fritekstId]: redigerbarefritekster[fritekstId].valider({
+                                        ...redigerbarefritekster[fritekstId],
+                                        verdi: event.target.value,
+                                    }),
+                                });
+                            }}
+                        />
+
+                        <SletteKnapp
+                            erLesevisning={erLesevisning()}
+                            onClick={() => {
+                                delete redigerbarefritekster[fritekstId];
```

Comment:
Dette er en litt uheldig bruk av mutering. Prøv å skriv om til noe ala:
```suggestion
                                settRedigerbarefritekster([...redigerbarefritekster.filter(fritekst => fritekst.id !== fritekstId)]);
```

Reply:
Denne slet jeg mye med prøvde bla det foreslagte løsningen, men den kompilerer dessverre ikke.